src/components/Lend/LoanSearch/LoanSearchInterface.vue

```diff
@@ -1,15 +1,16 @@
 <template>
-	<div class="tw-flex">
-		<div id="mobile-menu" class="tw-rounded md:tw-hidden
-				tw-z-dropdown tw-drop-shadow tw-absolute tw-top-4 md:tw-top-0"
-		>
-			<loan-search-filter />
-		</div>
+	<div class="tw-flex" @keyup.esc="closeLightbox">
 		<div class="tw-flex tw-flex-col tw-mr-4">
 			<div class="md:tw-hidden tw-mb-3">
-				<kv-button variant="secondary">
+				<kv-button variant="secondary" @click="openLightbox">
 					Filter & Sort
 				</kv-button>
+
+				<kv-lightbox :visible="isLightboxVisible" title="" preventClose
```

Comment:
We shouldn't need the `preventClose` prop here. There's no reason to prevent closing the filter menu